scripts/plot_levels_of_automation.py

```diff
@@ -0,0 +1,105 @@
+from typing import List
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib import rcParams
+
+plt.style.use("seaborn-pastel")
+rcParams.update(
+    {
+        "font.family": ["Helvetica", "sans-serif"],
+        "axes.labelsize": 18,
+        "xtick.labelsize": 14,
+        "ytick.labelsize": 14,
+        "xtick.major.size": 7,
+        "ytick.major.size": 7,
+        "xtick.major.width": 2.0,
+        "ytick.major.width": 2.0,
+        "font.size": 18,
+        "axes.linewidth": 2.0,
+        "lines.dashed_pattern": (5, 2.5),
+        "legend.edgecolor": "black",
+        "lines.markersize": 8,
+        "lines.linewidth": 2.0,
+    }
+)
+
+AUTOMATED = np.array([2, 1, 1, 1, 1, 9, 1])
```

Comment:
Please add a description/explanation of this variable. Perhaps even with a simpler example/reference to the paper.